plugins/bgnotify/bgnotify.plugin.zsh

```diff
@@ -62,7 +62,7 @@ function bgnotify_formatted {
 function bgnotify_appid {
   if (( ${+commands[osascript]} )); then
     osascript -e "tell application id \"$(bgnotify_programid)\"  to get the {id, frontmost, id of front window, visible of front window}" 2>/dev/null
-  elif [[ -n $WAYLAND_DISPLAY ]] && (( ${+commands[swaymsg]} )); then # wayland+sway
+  elif [[ -n $WAYLAND_DISPLAY ]] && [[ -n $SWAYSOCK ]] && (( ${+commands[swaymsg]} )); then # wayland+sway
```

Comment:
Looks like `swaymsg` can also fallback to the value of `$I3SOCK` if `$SWAYSOCK` is unset.

Reply:
Indeed, let's change it.